.vscode/launch.json

```diff
@@ -10,15 +10,23 @@
             "request": "launch",
             "mainClass": "us.dot.its.jpo.ode.aws.depositor.AwsDepositor",
             "projectName": "jpo-aws-depositor",
-            "args": ["--bootstrap-server", "<kafka host>:9092",
-                    "-d", "s3",
-                     "-g", "testgroup",
-                     "-k", "testkey",
-                     "-b", "s3dtestbucket",
-                     "-t", "test"],
+            // arguments -i, -a, -n, -e, & -h are optional and not used in this launch configuration
+            // replace <docker_host_ip> with the IP address of the docker host
+            // replace <access key id> with the access key id of the AWS account
```

Comment:
For these secret level args, can we put those in a .env file and read from that here, then just update the README to reflect? I'm concerned that someone will inevitably check in their credentials.

Reply:
I don't think we're able to reference environment variables directly in the "args" section of the launch configuration, so if we were to get rid of the `-x` option, that would result in the program failing since that argument is required.